docs/using/sdk_api.md

```diff
@@ -1,3 +1,3 @@
 # `hyp3_sdk` *v1.4.1* API Reference
 
-::: hyp3_sdk
+{{ get_content('https://raw.githubusercontent.com/ASFHyP3/hyp3-sdk/v1.4.1/README.md') }}
```

Comment:
Likewise, this change make it so our SDK API reference:
https://hyp3-docs.asf.alaska.edu/using/sdk_api/

renders as just our SDK README.md:
https://hyp3-docs.asf.alaska.edu/using/sdk/

and again, if we really can't get mkdocstrings to work, it'd be better to eliminate this API reference page entirely.